install.sh

```diff
@@ -136,7 +180,7 @@ function download_check () {
     fi
     #download
     echo -e "${BLUE}Downloading ${1}...${RESET}"
-    $CURL '-#' -L "${2}" -o "${3}"
+    curl '-#' -L "${2}" -o "${3}"
```

Comment:
Why was this line changed?  Don't we want to use the environment variable instead?

Reply:
The `${CURL}` variable is already applied to the `curl` command in `function curl()`:
https://github.com/skycocker/chromebrew/blob/ecfc304e46e082c5f2490f097f5f2acf1e86977c/install.sh#L74-L79